[Content] More Landable Gas Giants (#8274)
endless-sky/endless-sky: data/map planets.txt

	"required reputation" 15
	security 0.35

planet "Deka Dathnak"
	attributes "gas giant" "requires: gaslining" uninhabited
	landscape land/nasa21
	description `For whatever reason, someone has gone to the trouble of bombarding this gas giant with thousands of nuclear warheads. Swirls of radioactive isotopes are plainly visible in the lower atmosphere, and the amount of soot floating around points towards a large amount of some material having been incinerated in the recent past. The Korath were presumably the ones who decided to drop several gigatons worth of nuclear weapons into this gas giant during their civil war, but the exact reason why they did so remains unknown.`

> 	description `	Unfortunately there is no eerie green radioactive glow to illuminate the layers beneath the clouds. Instead, your ship is shrouded in shadow and ash, plunging your viewports into darkness.`

Reviewer: At least provide a reason for why you're dismissing these suggestions.

	government Uninhabited
	security 0

planet Jupiter
	attributes "gas giant" "requires: gaslining" uninhabited
	landscape land/nasa12

> 	description `Once called the King of the Solar System, Jupiter is just as impressive even considering the variety of gas giants in the galaxy. Its famous Great Red Spot is still well-known by most of humanity, and you make sure to steer clear of it while taking in the rich yellows and browns of the planet's atmosphere.`

Reviewer 2: Yeah, I'm not sure about that either. I just thought I should mention it.

> 	description `Once called the King of the Solar System, Jupiter is just as impressive even considering the variety of gas giants in the galaxy. Its famous Great Red Spot is still well-known by most of humanity, and you make sure to steer clear of it while taking in the rich yellows and browns of the planet's atmosphere.`

Reviewer 3: The Great Red Spot is visible in Jupiter's planet sprite, so in that sense it's canon to ES that it's still around, whether or not that's realistic.

> 	description `Once called the King of the Solar System, Jupiter is just as impressive even considering the variety of gas giants in the galaxy. Its famous Great Red Spot is still well-known by most of humanity, and you make sure to steer clear of it while taking in the rich yellows and browns of the planet's atmosphere.`

Reviewer 4: Jupiter may have grown a new storm which is similarly red, who knows?

> 	description `Once called the King of the Solar System, Jupiter is just as impressive even considering the variety of gas giants in the galaxy. Its famous Great Red Spot is still well-known by most of humanity, and you make sure to steer clear of it while taking in the rich yellows and browns of the planet's atmosphere.`

Reviewer 5: I actually think it would be funny if it would be gone.
Showing that the contributors of the game actually take real life data into account(which we do not)

	description `Aside from fish, the Kimek diet is mostly vegetarian, but a few centuries ago an Arach entrepreneur decided to start ranching longcows here. It is an ideal world for ranching, with plenty of grassy fields. The ranches have begun a major industry, although generally only the wealthiest of Kimek are willing to spend money on longcow meat.`
	spaceport `The spaceport is at the very center of a city that is laid out with perfect radial symmetry: six sectors, each with their own markets, schools, and other public services, and clusters of skyscrapers that can each house a million individuals. The spaceport itself is also symmetrical, with a concourse in the center and smaller and smaller landing pads radiating out from it so that the largest freighters have easiest access.`

planet Nepheritis
	attributes "gas giant" "requires: gaslining" uninhabited
	landscape land/clouds_08

> 	description `An unusually large ice giant, Nepheritis is well-known among the denizens of the Paradise Planets for its prodigious rains of large diamonds. While this phenomenon is common on most ice giants of sufficient size, Nepheritis became a scientific curiosity due to the sheer size of the diamonds formed in its atmosphere, with some even becoming large boulders during their plummet into the depths.`

Reviewer: This suggestion only covers the weird tense shift in the middle.
```suggestion
	description `An unusually large ice giant, Nepheritis is well-known among the denizens of the Paradise Planets for its prodigious rains of large diamonds. While this phenomenon is common on most ice giants of sufficient size, Nepheritis is a scientific curiosity due to the sheer size of the diamonds formed in its atmosphere, with some even becoming large boulders during their plummet into the depths.`
```

	description `Aside from fish, the Kimek diet is mostly vegetarian, but a few centuries ago an Arach entrepreneur decided to start ranching longcows here. It is an ideal world for ranching, with plenty of grassy fields. The ranches have begun a major industry, although generally only the wealthiest of Kimek are willing to spend money on longcow meat.`
	spaceport `The spaceport is at the very center of a city that is laid out with perfect radial symmetry: six sectors, each with their own markets, schools, and other public services, and clusters of skyscrapers that can each house a million individuals. The spaceport itself is also symmetrical, with a concourse in the center and smaller and smaller landing pads radiating out from it so that the largest freighters have easiest access.`

planet Nepheritis
	attributes "gas giant" "requires: gaslining" uninhabited
	landscape land/clouds_08

> 	description `An unusually large ice giant, Nepheritis is well-known among the denizens of the Paradise Planets for its prodigious rains of solid diamonds. While this phenomenon is common on most ice giants of sufficient size, Nepheritis became a scientific curiosity due to the sheer size of the diamonds formed in its atmosphere, with some reaching the size of large boulders before plummeting into its depths.`

Reviewer 3: Not sure about this - I'd say the original fits better for this being the information you get when landing on Nepheritis (the primary sentence is "Nepheritis is well-known among the denizens of the Paradise Planets for its prodigious rains of solid diamonds", which is describing the planet.
Your revision makes the name a minor clarification, which would fit better if it was being mentioned in passing (in spaceport news, for example), but doesn't really fit as well when you're landed on the planet.

	spaceport `The spaceport is at the very center of a city that is laid out with perfect radial symmetry: six sectors, each with their own markets, schools, and other public services, and clusters of skyscrapers that can each house a million individuals. The spaceport itself is also symmetrical, with a concourse in the center and smaller and smaller landing pads radiating out from it so that the largest freighters have easiest access.`

planet Nepheritis
	attributes "gas giant" "requires: gaslining" uninhabited
	landscape land/clouds_08
	description `An unusually large ice giant, Nepheritis is well-known among the denizens of the Paradise Planets for its prodigious rains of large diamonds. While this phenomenon is common on most ice giants of sufficient size, Nepheritis became a scientific curiosity due to the sheer size of the diamonds formed in its atmosphere, with some even becoming large boulders during their plummet into the depths.`

> 	description `	While these diamonds are unfortuntely not economical to collect with current human technology, if you were so inclined you could perhaps make an attempt at catching one as it fell, deep within the atmosphere where they are largest.`

Reviewer 6: ```suggestion
	description `	While these diamonds are unfortunately not economical to collect with current human technology, if you were so inclined you could perhaps make an attempt at catching one as it fell, deep within the atmosphere where they are largest.`
```

	description `Very similar to its sister station of Seli'het, Parii'het was still under construction when it was abandoned by its creators. Today, it floats uninhabited in this rarely visited section of space, approached occasionally by the automated drones that mindlessly continue to carry new supplies to the massive cargo section. If the Ka'het attacked this station, it would be made entirely inoperable in a short time.`
	government Builder

planet Passaritus
	attributes "gas giant" "requires: gaslining" uninhabited
	landscape land/nasa27

> 	description `Decades ago, Passaritus was the subject of an ambitious project by the Deep to economically mine metallic hydrogen from its mantle. Being barely large enough to form metallic hydrogen in the first place, the gas giant's gravity was thought to be low enough to make extraction feasible. Unfortunately, Deep engineers overestimated their ability to build an extraction device that was not only technically workable, but economical, and the project has since been moved to the backburner in favor of more reasonable endeavors.`

Reviewer 6: ```suggestion
	description `Decades ago, Passaritus was the subject of an ambitious project by the Deep to economically mine metallic hydrogen from its mantle. Being barely large enough to form metallic hydrogen in the first place, the gas giant's gravity was thought to be low enough to make extraction feasible. Unfortunately, the Deep's engineers overestimated their ability to build an extraction device that was not only technically workable, but economical, and the project has since been moved to the backburner in favor of more reasonable endeavors.`
```

planet Pyriphlegethon
	attributes "gas giant" "requires: gaslining" uninhabited
	landscape land/clouds_02

> 	description `Despite its poorly received name, Pyriphlegethon was famous in the early history of human interstellar travel, as the first "hot Jupiter" to be explored in detail. Colorful images of its incandescent clouds and glowing night sky became some of the most iconic images of space in the 23rd century, when the invention of the hyperdrive was still recent.`

Reviewer 7: ```suggestion
	description `Despite its poorly received name, Pyriphlegethon was famous in the early history of human interstellar travel as the first "hot Jupiter" to be explored in detail. Colorful images of its incandescent clouds and glowing night sky became some of the most iconic images of space in the 23rd century, when the invention of the hyperdrive was still recent.`
```

	landscape land/desert7
	description `This was probably a habitable world at one point. It is no longer. Nothing lives or grows here, and the atmosphere is full of toxic chemicals.`

planet "Sebarep Atarkis"
	attributes "gas giant" "requires: gaslining" uninhabited
	landscape land/clouds_04

> 	description `A recent event, natural or artificial, disrupted the rings around Sebarep Atarkis to the point of complete instability. Now, hundreds of ringroids fall into the planet's atmosphere each day, tracing fiery paths near the equator. This has measurably increased the temperature in the equatorial region, and new storms and wind patterns are generated chaotically.`

Reviewer 6: What about moonlets? https://en.wikipedia.org/wiki/Moonlet

	"required reputation" 15
	security 0.15

planet "Shih's Locker"
	attributes "gas giant" "requires: gaslining" uninhabited
	landscape land/nasa16

> 	description `Centuries ago, this gas giant earned its name after the eponymous and infamous Captain Shih "disposed" of a mutiny by ejecting most of her crew into its atmosphere. Cruelly, they were packed into an escape pod that survived at least the upper atmosphere of her Locker, instead of simply being tossed out of an airlock as was the standard practice. In the years that followed, many seeking to prove themselves the meanest pirate in the galaxy sought to emulate her example by making the pilgrimage to her Locker whenever a crewmember went too far out of line.`
> 	description `	Cold clouds of ammonia swirl around your ship as it dives deeper. This far within the planet's atmosphere, the sky is oddly colored a murky blue-green, which stands in contrast to the brownish haze that seems to extend for eternity. It is not at all difficult to imagine yourself at the bottom of a terrestrial planet's ocean. While legend on Greenrock holds that the ghosts of traitorous pirates haunt the depths of Shih's Locker, they seem to be keeping to themselves for now.`

Reviewer: ```suggestion
	description `This gas giant earned its name after the infamous Captain Shih "disposed" of a mutiny. Rather than simply ejecting her crew out of the airlock, she packed them into an escape pod that would survive the upper atmosphere of the planet. In the years that followed, many seeking to prove themselves the meanest pirate in the galaxy sought to emulate her example by making a pilgrimage to her Locker whenever a crewmember went too far out of line.`
	description `	This far within the planet's atmosphere, the sky is colored a murky blue-green, standing in contrast to the brownish haze that seems to extend for eternity. While legend on Greenrock holds that the ghosts of traitorous pirates haunt the depths of Shih's Locker, they seem to be keeping to themselves for now.`
```

	"required reputation" 15
	security 0.5

planet Steamwater
	attributes "gas giant" "requires: gaslining" uninhabited
	landscape land/clouds_05

> 	description `Steamwater is a hot Neptune, a misleadingly named "ice giant" close to its parent star. Steamwater is distinctive in that, by mass, it is almost entirely water. Beyond this, the range of temperatures and pressures found in its atmosphere is such that its thickest layer is composed of supercritical water, a phase state blending together the states of vapor and liquid, making flight inside it rather disorienting.`

Reviewer: ```suggestion
	description `Steamwater is a hot Neptune, a misleadingly named "ice giant" close to its parent star. Steamwater is distinctive in that it is almost entirely water by mass. Beyond this, the range of temperatures and pressures found in its atmosphere is such that its thickest layer is composed of supercritical water, blending together the states of vapor and liquid, making flight inside it rather disorienting.`
```

> 	description `	With no true interface between gaseous water vapor and the compressed liquid found below, the buoyant force affecting your ship is unpredictable without keeping a careful eye on your altimeter. While this is true of gas giants in general, the much higher density of water as compared to common hydrogen makes this effect especially noticeable.`

Reviewer 8: It couldn't actually be an altimeter, no existing form of altimeter would be able to handle buoyancy effects or an inconclusive phase state boundary.
Inertial navigation would use gyroscopes and accelerometers to track where you're going relative to where you were.
```suggestion
	description `	With no true interface between gaseous water vapor and the compressed liquid found below, the buoyant force affecting your ship is unpredictable without keeping a careful eye on your inertial navigation aid. While this is true of gas giants in general, the much higher density of water as compared to common hydrogen makes this effect especially noticeable.`
```

Reviewer 8: Yeah, but this isn't some fancy future tech. Inertial navigation is technology we've been using for many years already in the real world.

Reviewer 9: As an average person, I would understand what an "altimeter" is referring to and that sentence makes sense to me. 
"inertial navigation aid" sounds like nonsense which my eyes just kinda glaze over and the rest of the sentence doesn't make sense as a result.

Reviewer 8: I suppose in a theoretical future with space travel, an "altimeter" could have become a device that incorporates such things.
Given the type of player who's actually likely to fly around in a gaslined ship and find this though, I'd suggest still at least calling it an "inertial altimeter".
```suggestion
	description `	With no true interface between gaseous water vapor and the compressed liquid found below, the buoyant force affecting your ship is unpredictable without keeping a careful eye on your inertial altimeter. While this is true of gas giants in general, the much higher density of water as compared to common hydrogen makes this effect especially noticeable.`
```

> 	description `	With no true interface between gaseous water vapor and the compressed liquid found below, the buoyant force affecting your ship is unpredictable without keeping a careful eye on your inertial altimeter. While this is true of gas giants in general, the much higher density of water as compared to common hydrogen makes this effect especially noticeable.`

Reviewer: ```suggestion
	description `	With no true transition between the gaseous water vapor and the compressed liquid found below, the buoyant force affecting your ship is unpredictable without keeping a careful eye on your inertial altimeter. While this is true of gas giants in general, the much higher density of water as compared to common hydrogen makes this effect especially noticeable.`
```

	bribe 0
	security 0

planet "Weh Mar"
	attributes "gas giant" "requires: gaslining" uninhabited
	landscape land/clouds_05

> 	description `Wah Mar is a hot Neptune, a misleadingly named "ice giant" close to its parent star. Usually composed mostly of methane, Wah Mar is distinctive in that, by mass, it is almost entirely water. Beyond this, the range of temperatures and pressures found in its atmosphere is such that its thickest layer is composed of supercritical water, making flight inside it rather disorienting.`

Reviewer 3: ```suggestion
	description `Wah Mar is a hot Neptune, a misleadingly named "ice giant" close to its parent star. While similar planets are usually composed mostly of methane, Wah Mar is distinctive in that, by mass, it is almost entirely water. Beyond this, the range of temperatures and pressures found in its atmosphere is such that its thickest layer is composed of supercritical water, making flight inside it rather disorienting.`
```

> 	description `Whether it be from local evolution or panspermia, Vivid Aether's clouds are home to colonies of anaerobic microbial life, feeding on both sunlight and traces of methane in the atmosphere. These form colorful streaks in the upper atmosphere, which initially attracted Saryd explorers to the planet. Now, it is common for Heliarch research patrols to sit in orbit for weeks at a time in order to study the simple, yet exotic, ecology. Occasionally, they send down hardened research vessels capable of withstanding the extreme pressures, though none happen to be nearby right now.`

Reviewer: ```suggestion
	description `Whether it be from local evolution or panspermia, Vivid Aether's clouds are home to colonies of anaerobic microbial life, feeding on both sunlight and traces of methane in the atmosphere. These form colorful streaks in the upper atmosphere, which attracted Saryd explorers to the planet. Now, it is common for Heliarch research patrols to sit in orbit for weeks at a time in order to study the simple, yet exotic, ecology. Occasionally, they send down hardened research vessels capable of withstanding the extreme pressures, though none happen to be nearby right now.`
```

Reviewer: That is already implicit through the use of "attracted" instead of "attracts".

> 	description `	The life-bearing layer only extends downward a few hundred kilometers before pressures rise and the temperature becomes inhospitable to life. Deeper within the planet's atmosphere, the environment is barren and lifeless.`

Reviewer: ```suggestion
	description `	The life-bearing layer only extends downward a few hundred kilometers before pressures rise and the temperature becomes inhospitable to life, giving way to a barren and lifeless environment.`
```
